libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -742,8 +768,9 @@ bool AP_DDS_Client::create()
         .id = 0x01,
         .type = UXR_PARTICIPANT_ID
     };
-    const char* participant_ref = "participant_profile";
-    const auto participant_req_id = uxr_buffer_create_participant_ref(&session, reliable_out, participant_id,0,participant_ref,UXR_REPLACE);
+    const char* participant_name = "ardupilot_dds";
+    const auto participant_req_id = uxr_buffer_create_participant_bin(&session, reliable_out, participant_id,
+                                    static_cast<uint16_t>(domain_id), participant_name, UXR_REPLACE);
```

Comment:
```suggestion
    const char* participant_ref = "participant_profile";
    const auto participant_req_id = uxr_buffer_create_participant_ref(&session, reliable_out, participant_id,
                                    static_cast<uint16_t>(domain_id), participant_ref, UXR_REPLACE);
```

Bad merge, will fix.